### PR TITLE
Date of promoted comment to answer adjusted

### LINKED ISSUE #2099: Adjust date of comments that are changed to answers on /questions
A new feature was added that allows people to move comments on /questions to answers. However, when comments are now adjusted, a new email goes out and the date of the original post is not maintained. 

Here's a link to the code: https://github.com/publiclab/plots2/blob/ceb7d05404b4619c7fa8cd8cae5e8215798cb02c/app/controllers/comment_controller.rb#L154-L157

### PR DESCRIPTION
**Pull request by Gauravano**
Fixes #2099 

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

File: test/functional/comment_controller_test.rb [hunk 1]
```diff
@@ -315,7 +315,7 @@ def teardown
           id: comment.id
     end
     assert_not_nil :answer
-    assert_not_equal initial_mail_count, ActionMailer::Base.deliveries.size
+    assert_equal initial_mail_count, ActionMailer::Base.deliveries.size
```
Comment (jywarren) [inline test/functional/comment_controller_test.rb]: oh, so this then wouldn't trigger a new notification, right? Maybe we should add a comment here explaining that? 

Thank you!

Comment (Gauravano) [reply test/functional/comment_controller_test.rb]: yes, its test that no email is generated on the promotion of comment to answer.I will add a comment explaining this or I can remove this check what you say @jywarren?

**Comment by jywarren** [comment]
that sounds great -- go ahead and add the comment, it's useful! Thanks!!!

**Comment by Gauravano** [comment]
@jywarren comments are added with tests.Please see https://github.com/publiclab/plots2/issues/2099#issuecomment-362930143 Thanks.

**Comment by jywarren** [comment]
Awesome. Thank you!!!

**Pull request merged**